auth/src/main/java/org/openedx/auth/presentation/signin/compose/SignInView.kt

```diff
@@ -264,15 +266,15 @@ private fun AuthForm(
                         onEvent(AuthEvent.ForgotPasswordClick)
                     },
                 text = stringResource(id = R.string.auth_forgot_password),
-                color = MaterialTheme.appColors.primary,
+                color = MaterialTheme.appColors.info_variant,
                 style = MaterialTheme.appTypography.labelLarge
             )
         }
 
         if (state.showProgress) {
             CircularProgressIndicator(color = MaterialTheme.appColors.primary)
         } else {
-            OpenEdXButton(
+            OpenEdXSecondaryButton(
                 modifier = buttonWidth.testTag("btn_sign_in"),
                 text = stringResource(id = coreR.string.core_sign_in),
                 onClick = {
```

Comment:
This solution causes a lot of confusion. Why do we use a Secondary Button for the main action on the screen?
Please do not introduce a new button type, use styling for the existing one.